### ISSUE TITLE
Pyreverse fails if the source uses the walrus operator (:=)

### ISSUE THREAD

**Issue by NoelJB** [PyCQA/pylint#3627]
# Steps to reproduce
1. create a file using the walrus operator, e.g.,
`if msg:= "QED":
    print(msg)
`
2. pyreverse3 proof.py

### Current behavior
$  pyreverse3 proof.py 
parsing proof.py...
Parsing Python code failed:
invalid syntax (<string>, line 1)

### Expected behavior
$  pyreverse3 proof.py 
parsing proof.py...
$ 

### pylint --version output
$  pylint3 --version
No config file found, using default configuration
pylint3 1.8.3, 
astroid 1.6.0
Python 3.6.9 (default, Apr 18 2020, 01:56:04) 
[GCC 8.4.0]

**Comment by PCManticore** [comment]
The walrus operator is a Python 3.7 (or 3.8?) specific feature and you seem to be using 3.6.9. Other than that, pylint 1.8.X does not support that feature, you would have to upgrade to 2.4 or 2.5.

**Comment by NoelJB** [comment]
Actually, I have about 6 versions of Python installed, including Python 3.  I had tried running the preview version of pylint without seeing any difference, but I've got it working now.  What I found is that it matters which version of python you run pylint using in order for it to process the walrus operator.  I hadn't expected that, but OK.  Ensuring that the current version of pylint and pyreverse are running under Python 3.8 solved the issue.
